### PR TITLE
[Data Visualizer] Add Bar Chart to modal

### PR DESCRIPTION
**Pull request by ajpal**
# Description
If Bar Chart is the selected `chartType`, and a column has been selected from the Values dropdown, then render a Google MaterialBarChart. There is a bar for each value in the selected column, and the height of the bar is the number of times that value appears in the column.
image

## Links

- [spec](https://docs.google.com/document/d/1s1CQ4SSrpwJtLwI5gJoFzJEbcEXZHQIQ6ikAxB9H3j4/edit?usp=drive_web&ouid=106447758460192126987)
- [jira](https://codedotorg.atlassian.net/browse/STAR-752)

## Testing story
No tests yet. I'll write tests when the full modal is complete, because I have a feeling things will change a bit.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/storage/dataBrowser/DataVisualizer.jsx [hunk 1]
```diff
@@ -36,7 +37,37 @@ class DataVisualizer extends React.Component {
     this.setState({isVisualizerOpen: false});
   };
 
+  aggregateRecordsByColumn = (records, columnName) => {
```
Comment (jmkulwik) [inline apps/src/storage/dataBrowser/DataVisualizer.jsx:40]: Haha this looks like an interview question :)

**Review by jmkulwik — APPROVED** [review]
LGTM!

File: apps/src/storage/dataBrowser/DataVisualizer.jsx [hunk 2]
```diff
@@ -36,7 +37,37 @@ class DataVisualizer extends React.Component {
...
+    let counts = {};
+    records.forEach(record => {
+      let value = record[columnName];
+      counts[value] = (counts[value] || 0) + 1;
+    });
```
Comment (islemaster) [inline apps/src/storage/dataBrowser/DataVisualizer.jsx:45]: Consider [`_.countBy`](https://lodash.com/docs/4.17.15#countBy)
```js
const counts = _.countBy(records, r => r[columnName]);
```

File: apps/src/storage/dataBrowser/DataVisualizer.jsx [hunk 3]
```diff
@@ -36,7 +37,37 @@ class DataVisualizer extends React.Component {
...
+    let chartData = [];
+    Object.keys(counts).forEach(key => {
+      chartData.push({[columnName]: key, count: counts[key]});
+    });
```
Comment (islemaster) [inline apps/src/storage/dataBrowser/DataVisualizer.jsx:49]: Consider [`_.map`](https://lodash.com/docs/4.17.15#map)
```js
return _.map(counts, (count, key) => (
  {[columnName]: key, count}
));
```

**Review by islemaster — COMMENTED** [review]
Just a couple suggestions for a more concise aggregation helper.

**Pull request merged**